meson_options.txt

```diff
@@ -161,6 +161,9 @@ option('man', type : 'combo', choices : ['auto', 'true', 'false'],
 option('html', type : 'combo', choices : ['auto', 'true', 'false'],
        value : 'false',
        description : 'build and install html pages')
+option('translations', type : 'combo', choices : ['auto', 'true', 'false'],
```

Comment:
Why this is `combo`? It looks `boolean` is enough for this setting.

Reply:
I went with the same approach used by the html option which is also a combo. If a boolean is preferred I'll modify this to be a boolean.

Reply:
Switched to a boolean option.